node/src/components/linear_chain_sync/deploys.rs

```diff
@@ -0,0 +1,25 @@
+use crate::{
```

Comment:
I think the entire linear_chain_sync component has been removed on the feat-fast-sync branch.